buildtools/pipenv/pipenv_test.go

```diff
@@ -0,0 +1,109 @@
+package pipenv_test
+
+import (
+	"errors"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/fossas/fossa-cli/buildtools/pipenv"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+// newMockPipenv constructs a pipenv.PipenvCmd with a custom Graph function.
```

Comment:
[nit] `Mock constructs a pipenv.Cmd using mock build tool output.`

In general, declaration comments are more useful when they describe _why_ rather than _how_ a declaration is used.